tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import Button from '../Button'
+import { history } from '../../configureStore'
+import { setYear } from '../../actions/reports'
+
+/**
+ * @param {string} selectedYear = The year that the user has chosen from the
+ * Select component.
+ *
+ * Reports is the home page for users to file a report.
+ * The user can select a year
+ * for the report that they would like to upload and then click on
+ * `Begin Report` to begin uploading files for that year.
+ */
+function Reports() {
```

Comment:
This is the `Reports` component that is the home page for the `/reports` route. We use `selectedYear` from Redux in order to render the correct year on this page.